source/unified-test-format/unified-test-format.rst

```diff
@@ -3749,6 +3754,8 @@ Changelog
 ..
   Please note schema version bumps in changelog entries where applicable.
 
+:2022-10-07: **Schema version 1.12.**
```

Comment:
oh boy I get to do another rebase and schema version bump on #1303  🙂

Reply:
![](https://media.giphy.com/media/3orif9DJNKDfXqXbBS/giphy.gif)

In all seriousness, if your PR is nearly ready to merge I'm happy to hold off on this for a few more days and spare you the trouble.

Reply:
ha, I appreciate it, but it really isn't too big of a deal. I am really hoping my PR is going to get merged this week but I don't want to promise it. so let's just let things play out naturally timing wise